exist-core/src/main/java/org/exist/storage/NativeBroker.java

```diff
@@ -177,9 +173,9 @@
 
     private final Runtime run = Runtime.getRuntime();
 
-    private NodeProcessor nodeProcessor = new NodeProcessor();
+    private final NodeProcessor nodeProcessor = new NodeProcessor();
 
-    private IEmbeddedXMLStreamReader streamReader = null;
+    private final IEmbeddedXMLStreamReader streamReader = null;
```

Comment:
So `streamReader` will always be `null`?

seems that this should be removed, and `streamReaderNG` renamed to `streamReader`